crates/chia-consensus/src/run_block_generator.rs

```diff
@@ -244,3 +249,62 @@ where
     ret.cost = max_cost - cost_left;
     Ok(ret)
 }
+
+pub fn get_coinspends_for_trusted_block(
```

Comment:
do you think we could use the same test cases as we use for `run_block_generator2()` to unit test this function?

Reply:
Done